playwright.config.ts

```diff
@@ -15,7 +15,12 @@ const config: PlaywrightTestConfig = {
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      use: {
+        ...devices['Desktop Chrome'],
+        contextOptions: {
+          permissions: ['clipboard-read', 'clipboard-write'],
```

Comment:
I prefer if we don't require specific permissions. It should be feasible without them if you access the clipboard in a user-generated event (for example a click)

Reply:
It's possible to paste using clicks, but the function (copyToClipboard) need the permission in order to copy in js

Reply:
That's my point. I'd like to see the test simulate a user click and copy in the click handler, not directly call the `copyToClipboard` function. This should be possible without explicit permissions.

Reply:
> not directly call the `copyToClipboard` function.

Could you please explain this,I understood that we must test the function.
Otherwise if you want other html copy tests, I think it's possible

Reply:
You must test the function, but by calling it in a context where there was a user action (like `onClick`), not in a `useEffect`

Reply:
okey i get it thanks